### PR TITLE
[Mobile Payments] Remove duplicate back button from the Try Tap to Pay customer provided note screen

### LINKED ISSUE #12778: [Mobile Payments]Duplicate back arrow when editing default "customer provided note" when trying Tap to Pay
**Steps to reproduce**
1. Navigate to the payments hub menu (more menu -> Payments)'
2. Click on "Try Tap to Pay" option
3. Click on "Try payment" option
4. In this screen, try editing the customer provided note
5. Observe the duplicate back arrow button

https://github.com/user-attachments/assets/f964ff6e-2eac-44d5-b598-c84a1a646770

### PR DESCRIPTION
**Pull request by AnirudhBhat**
Closes: #12778 

### Description

This PR removes duplicate back button that appeared on the Try Tap to Pay customer provided note screen.

### Steps to reproduce

1. Navigate to the payments hub menu (more menu -> Payments)'
2. Click on "Try Tap to Pay" option
3. Click on "Try payment" option
4. In this screen, try editing the customer provided note
5. Observe the duplicate back arrow button

### The tests that have been performed

Tested on both dark and light mode, tested adding notes, removing notes.

### Images/gif

https://github.com/user-attachments/assets/cd6f5b94-54df-4a70-a261-a53dbeac8063

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [x] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [x] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [x] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on big (tablet) and small (phone) in case of UI changes, and no regressions are added.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/simplepayments/SimplePaymentsCustomerNoteFragment.kt [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
 import com.woocommerce.android.extensions.navigateBackWithResult
 import com.woocommerce.android.ui.base.BaseFragment
+import com.woocommerce.android.ui.main.AppBarStatus
 import org.wordpress.android.util.ActivityUtils
 import org.wordpress.android.util.DisplayUtils
 
 class SimplePaymentsCustomerNoteFragment :
-    BaseFragment(R.layout.fragment_order_create_edit_customer_note),
-    MenuProvider {
+    BaseFragment(R.layout.fragment_order_create_edit_customer_note) {
```
Comment (samiuelson) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/simplepayments/SimplePaymentsCustomerNoteFragment.kt:19]: 💡 I think we are not using any functionality provided by BaseFragment, so I suggest switching to the plain `Fragment` class.

**Review by samiuelson — APPROVED** [review]
Thanks for improving that @AnirudhBhat! I added a small suggestion—let me know what you think.

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/241">20.9</a>. The due date for this milestone has already passed.<br />
Please assign it to a milestone with a later deadline or check whether the release for this milestone has already been finished.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/simplepayments/SimplePaymentsCustomerNoteFragment.kt (hunk 1 above)
Comment (AnirudhBhat) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/simplepayments/SimplePaymentsCustomerNoteFragment.kt:19]: We are using `override val activityAppBarStatus: AppBarStatus
        get() = AppBarStatus.Hidden` from `BaseFragment` to hide the default toolbar and use our own. I think using plain Fragment would again cause multiple toolbar (one from our own custom toolbar and another that exist from previous screen)?

Comment (samiuelson) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/simplepayments/SimplePaymentsCustomerNoteFragment.kt:19]: I see, you're right 👍🏼

**Pull request merged**